src/Stryker.Core/Stryker.Core/Initialisation/InitialTestProcess.cs

```diff
@@ -28,21 +28,21 @@ public InitialTestProcess()
         public int InitialTest(ITestRunner testRunner)
         {
             _logger.LogInformation("Initial testrun started");
-
             // setup a stopwatch to record the initial test duration
             var stopwatch = new Stopwatch();
             stopwatch.Start();
 
-            var testResult = testRunner.RunAll(0, null);
-            _logger.LogInformation("Total number of tests found in initial test run: {0}", testResult.TotalNumberOfTests);
-
-            var duration = (int)stopwatch.ElapsedMilliseconds;
+            var testResult = testRunner.CaptureCoverage();
+            var duration = (int) stopwatch.ElapsedMilliseconds;
```

Comment:
Fair point, but there is actually little overhead with capture coverage. I pondered this myself and I see three options:
1. keep as is
2. have a specific test run. But this imply an extra run
3. use per test execution time as provided by VsTest. This would allow to compute per mutant timeout (by summing the run time of the impacted tests).

Anyway, I agree this is not a priority